modules/rostests/winetests/kernel32/testlist.c

```diff
@@ -1,5 +1,5 @@
 /* Automatically generated file; DO NOT EDIT!! */
-
+#include <R:\hook\kernel32_hk.h>
```

Comment:
I use this for testing locally, dunno how to omit it from the PR.

Reply:
Duno about other clients but magit can select which line to commit.
However, it's an Emacs plugin... (still awesome)

Reply:
In TortoiseGit, it's a bit awkward: Rightclick the file, choose "restore after commit", then open the diff, and remove that change.
I would say partial staging is the weakest point of tgit.

Reply:
Sure i can skip lines to commit, but i do not want unversioned files in my wc, because then i can no longer rebase

Reply:
That's what the stash is for, you can save the changes to the stash before rebasing, then pop the stash.

Reply:
I don't want to do that each time i commit or whatever